### ISSUE TITLE
"Internal server error on Hover: TypeMismatch" if service path has spaces or special characters

### ISSUE THREAD

**Issue by giacomoorsi** [jolie/vscode-jolie#10]
I'm developing a university project with Jolie. I have had problems in the Jolie extension for VS code. Every time I move the mouse on the code, the Output window opens with this error: 
![Error](https://user-images.githubusercontent.com/11297544/92956708-a5e0bf80-f467-11ea-88e3-0725fd770f75.jpeg)

I have macOS 10.15.5, java 14.0.2, jolie 1.9.1, and vscode-jolie 1.3.2
I have asked my colleges and it is a common problem with the extension. 

I've looked into the source code of the extension and I concluded that the problem is caused by 
`inspectPorts@Inspector( inspectionReq )( inspectionRes )`
on file `/server/src/internal/utils.ol:123` which throws a `FileNotFoundException` for the file in which the Inspector has to check the syntax. 

My project has a path with some spaces (`/Users/giacomoorsi/Documents/UNIBO/Secondo Anno/Sistemi Operativi/Jolie/Progetto/labso_pastaefajolie/`) and the URI given to the inspector is the following: `/Users/giacomoorsi/Documents/UNIBO/Secondo%20Anno/Sistemi%20Operativi/Jolie/Progetto/labso_pastaefajolie/`

Java throws a `FileNotFoundException` because the file with a path including `%20` doesn't exist. 

I've added the following lines of code to utils.ol just before `inspectPorts@Inspector( inspectionReq )( inspectionRes )` at line 123 in order to replace `%20` with a space in the URI string:
```
replacementRequest = regexResponse.group[5]
replacementRequest.regex = "%20"
replacementRequest.replacement = " "
replaceAll@StringUtils(replacementRequest)(inspectionReq.filename)

replacementRequest = inspectionReq.includePaths[1]
replaceAll@StringUtils(replacementRequest)(inspectionReq.includePaths[1])

```

This solution has solved the Hover issue and now the extension works perfectly. Note that my solution was tested only on macOS 10.15.5. It may cause new issues on other operative systems. 
I think `FileNotFoundException` may be thrown also if the URI contained a special character.

**Comment by fmontesi** [comment]
Looks good to me, thank you very much! Could you open a pull request with the fix?

@bmaschio could you test this on Windows?

**Comment by giacomoorsi** [comment]
> Looks good to me, thank you very much! Could you open a pull request with the fix?
> 
> @bmaschio could you test this on Windows?

Yes, I'll do it ASAP

**Comment by fmontesi** [comment]
Fixed by #11.
